dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -871,6 +872,7 @@ class BGPPeerMgrBase(Manager):
         :param key: key of the neighbor
         """
         vrf, nbr = self.split_key(key)
+        key = (vrf, nbr)
```

Comment:
Better to use different name here.

Reply:
@pavel-shirshov  updated to use "peer_key"